onnx/test/shape_inference_test.py

```diff
@@ -1183,6 +1183,21 @@ def test_squeeze(self) -> None:
             graph, [make_tensor_value_info("y", TensorProto.FLOAT, (3, 2))]
         )
 
+    def test_squeeze_no_axes_opset11(self) -> None:
+        graph = self._make_graph(
+            [
+                ("x", TensorProto.FLOAT, (1, 3, 1, 1, 2, 1)),
+            ],
+            [make_node("Squeeze", ["x"], "y")],
+            [],
+        )
+        operatorsetid = OperatorSetIdProto()
+        operatorsetid.domain = ""
+        operatorsetid.version = 11
+        self._assert_inferred(
+            graph, [make_tensor_value_info("y", TensorProto.FLOAT, (3, 2))]
```

Comment:
I guess that the opset needs to be specified, eg., something like `opset_imports=[helper.make_opsetid(ONNX_DOMAIN, 9)]`